.opspec/generate/op.yml

```diff
@@ -4,17 +4,16 @@ inputs:
   path:
     string:
       description: relative path to file containing a 'go:generate ...' pre-processing directive
-      default: ./cli/...
+      default: ./...
```

Comment:
What's the motivation for moving this out of cli project? It *seems* appropriately placed in the cli project. There is also one in the sdks/go project which *seems* appropriate as well. cli and sdk/go are separate projects; I might be working in one and want to `opctl run generate`; don't want to cd to root of repo to do that. Just my 2cents.

Reply:
It feels to me like there should be a root one, given that's where the go mod is. I can add one for the cli, no big deal.